### ISSUE TITLE
Export chapters have characters replaced with ! in urls.

### ISSUE THREAD

**Issue by DEMedus** [Mtillmann/chaptertool#11]
When I used the tool to create a chapter file I got unexpected results.
```
{
 "version": "1.2.0",
 "chapters": [
  {
   "startTime": 0,
   "img": "https!meduspod.com!ep!art!0024-600.jpg",
   "url": "https://meduspod.com",
   "title": "Intro"
  },
  {
   "startTime": 300,
   "img": "https!meduspod.com!assets!art!people!MaryKateUltra.jpg",
   "url": "https://meduspod.com",
   "title": "Chapter 2"
  }
 ]
}
``` 
The URLs for the images have ! points instead of the proper characters. It's a simple enough fix with find and replace, but thought I would let you know about it.

**Comment by Mtillmann** [comment]
Hi, this is very weird. Can you provide some context about when this happens and what the input was? Thanks

**Comment by DEMedus** [comment]
Sure, I went into the webapp. I started making my chapters. I always upload my chapters art to my server beforehand and pull each of them open so I have the url in the url bar of my browser. I simple copy and paste from the url bar to the image section of the webapp.
The urls for the images in my example should be
"https://meduspod.com/ep/art/0024-600.jpg" & "https://meduspod.com/assets/art/people/MaryKateUltra.jpg"
